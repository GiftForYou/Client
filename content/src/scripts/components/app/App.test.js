import React from 'react';
import {App} from './App';
const ReactShallowRenderer = require('react-test-renderer/shallow');
const renderer = new ReactShallowRenderer();

const component = renderer.render(<App/>);

test('[Content Test] overAll ', () => {
  expect(component).toMatchSnapshot();
});

test('[Content Test] type ', () => {
  expect(component.type).toMatchSnapshot();
});

test('[Content Test] props ', () => {
  expect(component.props).toMatchSnapshot();
});

test('[Content Test] store ', () => {
  expect(component._store).toMatchSnapshot();
});
