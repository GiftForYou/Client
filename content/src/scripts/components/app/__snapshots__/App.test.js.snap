// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Content Test] overAll  1`] = `<div />`;

exports[`[Content Test] props  1`] = `Object {}`;

exports[`[Content Test] store  1`] = `Object {}`;

exports[`[Content Test] type  1`] = `"div"`;
